src/components/AddBookForm.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/books/books';
+
+const AddBookForm = () => {
+  const dispatch = useDispatch();
+  const handleEvent = (e) => {
+    e.preventDefault();
+    const id = uuidv4();
+    const title = document.querySelector('.title-input').value;
+    const author = document.querySelector('.author-input').value;
+    dispatch(addBook(id, title, author));
+  };
```

Comment:
- Good Job in adding the local state with `title`, `author`. 👍 
- **Now, make sure they should have their inputs that update the local state every time they update** `(onChange)`